casper/src/main/rholang/MakePoS.rho

```diff
@@ -0,0 +1,111 @@
+contract @"MakePoS"(@purse, @minimumBond, @maximumBond, @initBonds, return) = {
+  new this, bondsCh, clonePurse, depositPurse in {
+    bondsCh!(initBonds) |
+  
+    //The purpose of this contract is to move the contents
+    //of an existing purse into a new purse. This prevents
+    //the owner of the given purse from modifying the balance
+    //during processing.
```

Comment:
Is this specifically because we don't process deploy sequentially?

Reply:
It's so that we don't check that the purse has the right balance at one step only later to see that the purse balance has changed. Basically, we move the authority from the user to the contract so that know once we verify the purse that it will not be changed suddenly.

Reply:
>  the purse has the right balance at one step only later to see that the purse balance has changed

So you're saying "yes" to my question right? If we processed deploys sequentially, then there is no way something can come in between the previous and next steps.